admin/links/class-link-watcher.php

```diff
@@ -55,6 +55,11 @@ public function save_post( $post_id, WP_Post $post ) {
 			return;
 		}
 
+		// When trashing a post
+		if ( $post->post_status === 'trash' ) {
```

Comment:
I would combine this statement with the statement above:
```php
$post_statuses_to_skip = array( 'auto-draft', 'trash' );
if( in_array( $post->post_status, $post_statuses_to_skip, true ) {
    return;
}
```

Reply:
Yes! That's more readable.

Reply:
Can you apply this suggestion?